galaxy_ng/app/api/ui/viewsets/execution_environment.py

```diff
@@ -210,3 +210,11 @@ class ContainerReadmeViewSet(ContainerContentBaseViewset):
     def get_object(self):
         distro = self.get_distro()
         return self.queryset.get_or_create(container=distro)[0]
+
+
+class SyncConfigContainerViewSet(
```

Comment:
Can this viewset be renamed to `ContainerRegistryRemoteViewSet` to more closely match the name of the model `ContainerRegistryRemote` and serializer `ContainerRegistryRemoteSerializer`?